### PR TITLE
Added migration to restore vewcheckdiagnostic

### PR DESCRIPTION
**Pull request by danwilliams**
- Whilst it isn't used in the codebase, it is used externally

### CODE REVIEW

File: admin/data/sql/migrations/20190614171658.do.restore-vewcheckdiagnostic.sql [hunk 1]
```diff
@@ -0,0 +1,30 @@
+CREATE VIEW [mtc_admin].[vewCheckDiagnostic] AS
```
Comment (jon-shipley) [inline admin/data/sql/migrations/20190614171658.do.restore-vewcheckdiagnostic.sql]: Please make this migration idempotent.  The view now exists in some environments and not others.  This view should always be the result.

Comment (danwilliams) [reply admin/data/sql/migrations/20190614171658.do.restore-vewcheckdiagnostic.sql]: Based on the fact that the original branch was merged prematurely, and the last commit ended up coming later... I have tried to establish the possible outcomes and hopefully addressed those now.

**Comment by GuyHarwood** [comment]
Tested and working with latest changes.

Will sign off once we have a green travis build

**Comment by danwilliams** [comment]
> Why are there so many migrations? I was only expecting to see one new up/down migration pair.

I thought you might have run them already, so I was playing safe. Amended to just the ones actually needed.

File: admin/data/sql/migrations/20190620172547.do.restore-vewcheckdiagnostic.sql [hunk 2]
```diff
@@ -0,0 +1,37 @@
+IF NOT EXISTS (
```
Comment (jon-shipley) [inline admin/data/sql/migrations/20190620172547.do.restore-vewcheckdiagnostic.sql]: Please turn this back to the standard 'create or alter view ...' statement without an 'EXEC'.

**Pull request merged**